docker-assets/README.md

```diff
@@ -34,14 +18,9 @@ The image has been tested and validated under docker-1.10.3 (Fedora24)
 The first time you run the container, it will initialize the database, **please allow 2-4 mins** for MIQ to respond.
 
 ```
-docker run --privileged -di -p 80:80 -p 443:443 manageiq/manageiq
-```
-Please note you can ommit some ports from the run command if you don't need to use them
-
-_**Note:**_ If you are running a RHEL family docker host you can now run the MIQ container unprivileged as recent versions of docker (1.10.3). Please ensure oci-systemd-hook and oci-register-machine packages are installed in your system.
-```
 docker run -di -p 80:80 -p 443:443 manageiq/manageiq
```

Comment:
Since 80 is no longer exposed, should this be changed?

Reply:
So, 80 technically is exposed because [the base image exposes it](https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app-frontend/Dockerfile#L57). And if you start the image like the the redirects from 80 to 443 will actually work correctly.

So I could go either way on this.